packages/client/src/utils/helpers.ts

```diff
@@ -154,3 +154,33 @@ export const calculateIntervalDuration = (interval: string | null) => {
     ? 1.5
     : 2;
 };
+/**
+ * interpolate text including <p/>
+ * @param template - String to be interpolated
+ * @returns string of interpolated text
+ */
+export const replaceHTMLTags = (template: string) => {
+  // eslint-disable-next-line no-irregular-whitespace
+  const regex = new RegExp(/( |<([^>]+)>)/gi);
+  return template.replaceAll(regex, "");
+};
+/**
+ * Format string of field value of email template by either surroduning it with {{ }} or an <a/> tag
+ * @param value - value that needs to be formatted to be inserted into the email template
+ * @returns formatted string
+ * */
+export const formatTemplateString = (value: string, message = "") => {
+  switch (value) {
+    case "icsFile":
+      return `<a href="{{ ${value} }}"> ${
+        message ||
+        "Clicca qui per aggiungere le tue prenotazioni al tuo calendario"
+      } </a>`;
+    case "bookingsLink":
+      return `<a href="{{ ${value} }}"> ${
+        message || "Clicca qui per prenotare e gestire le tue lezioni"
```

Comment:
I think the consensus is: they shouldn’t be translatable, but should be english by defult @silviot ?

Reply:
@delasous @ikusteu I think these can stay like this for now. A proper solution would be better, but it needs quite some planning and work. I think it could be worked on as an improvement, if needed.